src/src/carhm/hillel/Car.java

```diff
@@ -0,0 +1,19 @@
+package carhm.hillel;
+
+public class Car {
+
+    private void startElectricity() {
```

Comment:
looks good. 
Let's make it more interesting :) 
Let's add a field "model" which is a model of a car (e.g. "Mazda 3", "Audi A6" etc.)
Add constructor and getters + setters.
Add some output to startElectricity(), startCommand(), and startFuelSystem(). For example, "Audi A6: electricity is started". Please use String.format() to insert the model into the string.
Run it and check the output.